com.core447.StreamController.yml

```diff
@@ -0,0 +1,99 @@
+id: com.core447.StreamController
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: launch.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --share=network
+  - --socket=pulseaudio
+  - --device=all # Needed to communicate with the decks
+  - --talk-name=org.freedesktop.Flatpak # Allow plugins to run system commands
+  - --talk-name=org.freedesktop.UPower.*
+  - --talk-name=org.mpris.MediaPlayer2.*
```

Comment:
This should be `org.mpris.MediaPlayer2.appname.*`

Reply:
I need access to all players, because the play/pause/skip actions should not be limited to one player e.g. Spotify, but should work in all players.